src/lug/System/Exception.cpp

```diff
@@ -1,29 +1,61 @@
 #include <lug/System/Exception.hpp>
 #include <sstream>
 
+/**
```

Comment:
For the `brief` can you make better sentences? For example for the method `getFile` something like that "Get the name of the file where the exception was thrown."
For some parameters / return value, don't forget to say the word "name", not just "file" or "function" (or "line number" instead of "line"), that can be confusing for some users.